ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupViewDemoController.swift

```diff
@@ -66,15 +104,20 @@ class AvatarGroupViewDemoController: DemoController {
         settingsTitle.text = "Settings"
         container.addArrangedSubview(settingsTitle)
 
-        addRow(text: "Avatar count", items: [incrementBadgeButton, decrementBadgeButton], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+        let avatarCountButtonsStackView = UIStackView(arrangedSubviews: [incrementBadgeButton.view, decrementBadgeButton.view])
+        avatarCountButtonsStackView.spacing = 30
+        addRow(text: "Avatar count", items: [avatarCountButtonsStackView], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
 
         let backgroundColorSwitch = UISwitch(frame: .zero)
         backgroundColorSwitch.isOn = isUsingAlternateBackgroundColor
         backgroundColorSwitch.addTarget(self, action: #selector(toggleAlternateBackground(switchView:)), for: .valueChanged)
 
         addRow(text: "Use alternate background color", items: [backgroundColorSwitch], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
-        addRow(text: "Max displayed avatars", items: [maxAvatarsTextField, maxAvatarButton], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
-        addRow(text: "Overflow count", items: [overflowCountTextField, overflowCountButton], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+
+        maxAvatarButton.view.widthAnchor.constraint(equalToConstant: 50).isActive = true
```

Comment:
consolidate constraints calls in `viewDidLayoutSubviews` ?

Reply:
Wouldn't it potentially cause a new layout of the views?
There are other constraints being activated in other funcs like insertAvatarViews, so I believe if anything can be improved, it should be a separate change because not all constraints here are related to the button.